node/src/main.rs

```diff
@@ -205,15 +205,21 @@ async fn main() {
         let chain_head_update_listener = store_builder.chain_head_update_listener();
         let network_store = store_builder.network_store(idents);
 
-        let chains = Arc::new(networks_as_chains(
+        // To support the ethereum block ingestor, ethereum networks are referenced both by the
+        // `blockchain_map` and `ethereum_chains`. Future chains should be referred to only in
+        // `blockchain_map`.
+        let mut blockchain_map = BlockchainMap::new();
+        let ethereum_chains = networks_as_chains(
+            &mut blockchain_map,
```

Comment:
This hack is because I preferred to not refactor the block ingestor to use `BlockchainMap` since it is unlikely to be used for new chains, so `ethereum_chains` continues to exists.